xarray/core/types.py

```diff
@@ -33,3 +33,6 @@
 DaCompatible = Union["DataArray", "Variable", "DataArrayGroupBy", "ScalarOrArray"]
 VarCompatible = Union["Variable", "ScalarOrArray"]
 GroupByIncompatible = Union["Variable", "GroupBy"]
+
+ErrorChoice = Literal["raise", "ignore"]
```

Comment:
Great! We could find / replace for these at  some point

Reply:
Even though it is scope creep, I have typed any occurences that I have found in this PR.
Mypy is not complaining locally about these changes.